rosidl_default_generators/package.xml

```diff
@@ -18,6 +19,8 @@
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_cpp</buildtool_export_depend>
 
+  <group_depend>rosidl_typesupport_c_packages</group_depend>
+  <group_depend>rosidl_typesupport_cpp_packages</group_depend>
```

Comment:
same here between the group depends and the build_export_depends (applies to the other package.xml files as well)

Reply:
Since the following `buildtool_export_depend` are closely related to the `group_depend` I intentionally kept them "together".

Reply:
IMO it would make more sense to have all the typesupport packages grouped together (introspection grouped with the connext and opensplice ones) rather than the group_depends grouped with the introspection ts but separated from the vendor specific ts. It doesnt really matter it was just curious to have a split below but not here